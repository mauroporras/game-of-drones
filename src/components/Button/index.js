import React from 'react'

import './styles.css'

const Button = props => {
  return (
    <button
      className='Button'
      onClick={props.onClick}
      type={props.type}
    >
      {props.text}
    </button>
  )
}

export default Button
