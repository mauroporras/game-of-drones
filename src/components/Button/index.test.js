import React from 'react'
import renderer from 'react-test-renderer'

import Button from './index'

test("Component renders the correct text", () => {
  const component = renderer.create(
    <Button text='Go!' />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
