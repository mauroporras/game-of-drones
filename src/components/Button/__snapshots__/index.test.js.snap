exports[`test Component renders the correct text 1`] = `
<button
  className="Button"
  onClick={undefined}
  type={undefined}>
  Go!
</button>
`;
